content/en/data_jobs/dataproc.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Enable Data Jobs Monitoring for Spark on Dataproc - Google Cloud
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+[Data Jobs Monitoring][9] gives visibility into the performance and reliability of Apache Spark applications on Google Cloud Dataproc.
+
+## Requirements
+
+[Dataproc Release 2.0.x][10] or later is required. All of Debian, Rocky Linux, and Ubuntu Dataproc standard images are supported.
```

Comment:
Add the following requirement:

This guide is for **Dataproc clusters on Compute Engine**. If you are using **Dataproc on GKE**, refer to the [Kubernetes Installation Guide](https://docs.datadoghq.com/data_jobs/kubernetes/?tab=datadogoperator) instead.